src/main/kotlin/com/matthewrussell/trchunkbrowser/domain/I18N.kt

```diff
@@ -0,0 +1,213 @@
+package com.matthewrussell.trchunkbrowser.domain
+
+import com.jfoenix.controls.JFXButton
+import com.matthewrussell.trchunkbrowser.model.Language
+import javafx.beans.binding.Bindings
+import javafx.beans.binding.StringBinding
+import javafx.beans.property.ObjectProperty
+import javafx.beans.property.SimpleObjectProperty
+import javafx.beans.value.ObservableValue
+import javafx.scene.control.Button
+import javafx.scene.control.Label
+import javafx.scene.control.Tooltip
+import java.text.MessageFormat
+import java.util.*
+import java.util.Locale
+import java.util.concurrent.Callable
+
+
+/**
+ * I18N utility class..
+ *
+ */
+object I18N {
```

Comment:
What is the problem with tornadofx's i18n framework that you're trying to create your own?

I'm not opposed to a better library for doing localization, but this one won't scale; I don't like the idea of having to use it to create every UI widget we make, and add every language we support here when that's already given by which Messages_XX.properties files we have.

https://github.com/edvin/tornadofx/wiki/i18n
This talks about setting locales, and we should be able to add new by creating new Locale objects if they're not already in java. This also explains how to change the Locale with FX.locale.

Is the problem that text is not refreshing when you change the language?

Reply:
> Is the problem that text is not refreshing when you change the language?
@jsarabia Yes, that's why I added this library.